modules/representation_management/app/controllers/representation_management/v0/pdf_generator_base_controller.rb

```diff
@@ -133,6 +133,30 @@ def normalize_country_code_to_alpha2(country_code)
       def feature_enabled
         routing_error unless Flipper.enabled?(:appoint_a_representative_enable_pdf)
       end
+
+      def seed_organization_and_individual_in_dev
+        return unless Rails.env.development?
+
+        if AccreditedIndividual.count.zero?
+          FactoryBot.create(:accredited_individual,
+                            first_name: 'First',
+                            middle_initial: 'M',
+                            last_name: 'Last',
+                            address_line1: '789 Rep St',
+                            city: 'RepCity',
+                            state_code: 'RC',
+                            zip_code: '54321',
+                            phone: '5555555555',
+                            email: 'rep@rep.com',
+                            individual_type: 'claims_agent')
+        end
+        FactoryBot.create(:accredited_organization, name: 'Best VSO') if AccreditedOrganization.count.zero?
+
+        p '*' * 100, 'Seeded accredited individual and organization', '*' * 100,
+          "AccreditedIndividual: #{AccreditedIndividual.first.inspect}",
+          '*' * 100,
+          "AccreditedOrganization: #{AccreditedOrganization.first.inspect}"
+      end
```

Comment:
@jvcAdHoc How do you feel about this?

I put it here because I have to create dummy data now every time I work with the PDFs and my dev database is reset every time I bring the app down and run `make up` again.  I looked around and there are a few other `Rails.env.development?` checks inside controllers in the app.

Reply:
We should not seed data from inside a controller

Reply:
`my dev database is reset every time I bring the app down` this is interesting

Reply:
I yanked this for now.  It looks like `make up` runs `make db` before it starts the server and that wipes the database.  I'll get around that later.